### PR TITLE
Update a lot of images and add info about caja file manager

### PR DESCRIPTION
**Pull request by Jacalz**
# Description

This updates most images to the current Adapta theme and adds information about Caja file manager :+1:

### CODE REVIEW

File: common-tasks/managing-your-files/en.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-lastmod = "2017-03-11T00:40:25+02:00"
+lastmod = "2018-07-28T10:07:30+02:00"
 +++
 # Managing your files
 
-Solus provides software that enables you to store, organize, and view your files (such as documents, music, pictures, and videos).
 
-{{< altimg "nautilus.jpg" "help-center/common-tasks/managing-your-files/" >}}
\ No newline at end of file
+## Budgie and Gnome
```
Comment (JoshStrobl) [inline common-tasks/managing-your-files/en.md]: GNOME

File: common-tasks/managing-your-files/en.md [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+
+Solus Budgie and Gnome provides Nautilus, a file manager that enables you to store, organize, and view your files (such as documents, music, pictures, and videos).
```
Comment (JoshStrobl) [inline common-tasks/managing-your-files/en.md]: Gnome -> GNOME

File: common-tasks/managing-your-files/en.md [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+
+{{< altimg "nautilus.jpg" "help-center/common-tasks/managing-your-files/" >}}
+
+## MATE
+
+Solus Mate provides Caja, the file manager for the MATE desktop that enables you to store, organize, and view your files (such as documents, music, pictures, and videos).
```
Comment (JoshStrobl) [inline common-tasks/managing-your-files/en.md]: MATE

File: common-tasks/managing-your-files/en.md (hunk 1 above)
Comment (Jacalz) [reply common-tasks/managing-your-files/en.md]: Fixed with 
https://github.com/Jacalz/help-center-docs/commit/d647cc8ba2f87193fd04c15444fbb9f959368020

File: common-tasks/managing-your-files/en.md (hunk 2 above)
Comment (Jacalz) [reply common-tasks/managing-your-files/en.md]: Fixed with 
https://github.com/Jacalz/help-center-docs/commit/d647cc8ba2f87193fd04c15444fbb9f959368020

File: common-tasks/managing-your-files/en.md (hunk 3 above)
Comment (Jacalz) [reply common-tasks/managing-your-files/en.md]: Fixed with 
https://github.com/Jacalz/help-center-docs/commit/d647cc8ba2f87193fd04c15444fbb9f959368020

**Comment by Jacalz** [comment]
I have pushed the changes and this should thus be ready for a re-review

**Comment by JoshStrobl** [comment]
LGTM, thanks!

**Pull request merged**